integration-tests/common.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
I think this file can be removed - we don't need it in git and keep track of versions, do we?